resources/js/csrf.js

```diff
@@ -1,4 +1,4 @@
-var today = new Date();
+window.today = new Date();
```

Comment:
I am not sure about this, this was a predefined Laravel script, might be better to leave as is.